cts/cts-cli.in

```diff
@@ -352,6 +360,31 @@ class Test:
         return False
 
 
+class ValidatingTest(Test):
+    """A Test subclass that additionally runs test results through xmllint."""
+
+    def _validate_hook(self, rc, stdout, stderr, valgrind=False):
+        """Validate test output with xmllint."""
+        # If the test failed, don't bother validating its results.
```

Comment:
failures should still validate against the api-result schema. we might want to return the original error rather than the validation error though (or add something to the message indicating validation also failed)

Reply:
Fixed.